events/schema.py

```diff
@@ -471,6 +471,28 @@ def resolve_reference_id(self, info, **kwargs):
         return self.reference_id
 
 
+class TicketmasterEnrolmentNode(DjangoObjectType):
+    created_at = graphene.DateTime(required=True)
+
+    class Meta:
+        model = TicketSystemPassword
+        interfaces = (relay.Node,)
+        fields = ("event", "created_at")
+
+    def resolve_created_at(self, info):
+        return self.assigned_at
+
+
+class InternalOrTicketSystemEnrolmentUnion(graphene.Union):
```

Comment:
There isn't much of a precedent in the current code base but thinking about the naming of graphene.Union type classes. `EventOrEventGroup` is the only other class which is a Union and it is a union of EventNode and EventGroupNode. Some naming conventions come to mind:

| A naming convention | Union₁ | Union₂ |
|---|---|---|
| Current  | InternalOrTicketSystemEnrolmentUnion | EventOrEventGroup |
| Use Union suffix for all | InternalOrTicketSystemEnrolmentUnion | EventOrEventGroupUnion |
| Use Node suffix for all | InternalOrTicketSystemEnrolmentNode | EventOrEventGroupNode |
| Don't use any shared suffix | InternalOrTicketSystemEnrolment | EventOrEventGroup |

A consistent naming convention would ease code comprehension IMHO.

What do you think?

Reply:
I think I will go with `Union` suffix all the way!